### DISCUSSION TITLE
Add GitHub topics to this repo

### DISCUSSION THREAD

**Discussion by sagarreddypatil** [pmndrs/zustand#1944] [Ideas] [answered]
I can't believe it took me this long to find this, please add GitHub topics to this repo to make it easier to find

**Comment by dai-shi** [comment]
Sure. Would you give me suggestions of topics? Would be nice to do the same for https://github.com/pmndrs/jotai and https://github.com/pmndrs/valtio too.

**Reply by sagarreddypatil** [reply]
Here are a few off the top of my head
redux
react
reactjs
react-context
state-management
hooks

I have not used other libraries of yours, so I can't recommend any tags. But I suppose a nice way to go about it would be to find similar libraries and see how they're tagging their repos.

**Reply by dai-shi** [reply, marked as answer]
thanks! done!
